content/de/wallets/DogecoinWallet.md

```diff
@@ -0,0 +1,42 @@
++++
+type = "wallet"
+
+name = "Dogecoin Wallet"
```

Comment:
Langerhans' (Max's) wallet is no longer supported or updated, and has severe compatibility problems; I do not belive we should list it here, especially not being called 'sogood'.

Reply:
Did not know about issues, I use it a lot even on the latest Android 11 it works without any issues, but I will remove it :)

Reply:
i can double check with @langerhans but i'm pretty sure it hasn't been updated for 5 years and he doesn't wish to maintain it anymore.

Reply:
Yes, long time is not updated, but works really well even in my language Portuguese is the only wallet available in the World for Dogecoin lol